### PR TITLE
Add db migrations for rel 4.3

### PR DESCRIPTION
**Pull request by michalrak**
-New database model for operator's partners
-Premise and operator models update for UPRN
-Associated migrations

### CODE REVIEW

File: src/db/migrations/v1.8.1-add-partnership-details.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      },
+      operatorId: {
+        type: Sequelize.INTEGER,
+        references: {
+          model: "operators",
+          key: "id"
+        },
+        onUpdate: "cascade",
+        onDelete: "cascade"
+      }
```
Comment (off-beat) [inline src/db/migrations/v1.8.1-add-partnership-details.js]: All the other tables seem to have a created and updated date time field so I guess we should continue that - not sure how it is used but we can figure that out.

File: src/db/models/Operator.js [hunk 2]
```diff
@@ -24,7 +24,8 @@ module.exports = (sequelize, DataTypes) => {
       contact_representative_name: { type: DataTypes.STRING },
       contact_representative_role: { type: DataTypes.STRING },
       contact_representative_number: { type: DataTypes.STRING },
-      contact_representative_email: { type: DataTypes.STRING }
+      contact_representative_email: { type: DataTypes.STRING },
+      operator_uprn: { type: DataTypes.STRING }
```
Comment (off-beat) [inline src/db/models/Operator.js]: I think it's an integer, not a string - that was my mistake in the original API changes, sorry.

File: src/db/models/Premise.js [hunk 3]
```diff
@@ -11,7 +11,8 @@ module.exports = (sequelize, DataTypes) => {
       establishment_street: { type: DataTypes.STRING },
       establishment_town: { type: DataTypes.STRING },
       establishment_postcode: { type: DataTypes.STRING },
-      establishment_type: { type: DataTypes.STRING }
+      establishment_type: { type: DataTypes.STRING },
+      establishment_uprn: { type: DataTypes.STRING }
```
Comment (off-beat) [inline src/db/models/Premise.js]: again, this should be an integer

**Comment by off-beat** [comment]
> 
> 
> Looks good to me.
> 
> I just want to confirm, we are only having one set of contact details for the partnership, and that will be stored against the operator? If this is the case then I am happy with these changes.
> 
> The uprn changes look fine.

Yeah just one set of contact details against the operator - the user will see something slightly different

**Comment by cwcv** [comment]
OK. Happy to merge.

File: src/db/models/Premise.js (hunk 3 above)
Comment (off-beat) [reply src/db/models/Premise.js]: It sin integrer but FSA confirm treat as string

File: src/db/models/Operator.js (hunk 2 above)
Comment (off-beat) [reply src/db/models/Operator.js]: It is an integer but FSA confirm treat as string

**Pull request merged**